assets/js/modules/analytics/datastore/settings.js

```diff
@@ -299,3 +299,56 @@ export const isGA4DashboardView = createRegistrySelector( ( select ) => () => {
 
 	return ga4Settings.dashboardView === 'google-analytics-4';
 } );
+
+/**
+ * Determines whether the user should be prompted to switch to GA4 Dashboard View.
+ *
+ * @since n.e.x.t
+ *
+ * @return {boolean} True if the user should be prompted to switch to GA4 Dashboard, false otherwise, or undefined if not loaded.
+ */
+export const shouldPromptGA4DashboardView = createRegistrySelector(
+	( select ) => () => {
+		const isGA4Enabled = isFeatureEnabled( 'ga4Reporting' );
+		if ( ! isGA4Enabled ) {
+			return false;
+		}
+
+		const ga4ModuleConnected =
```

Comment:
Seeing as we have a boolean variable prefixed `is` directly above this, I'd suggest a rename here for consistency:

```suggestion
		const isGA4Connected =
```

Reply:
I wish we _did_ use is/has more regularly, but what you've suggested is consistent, so it LGTM!